scripts/k8s/deployer.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+SCRIPT_DIR="$(dirname "$0")"
+TMP_KIND="/tmp/kind/mayastor"
+TMP_KIND_CONFIG="$TMP_KIND/config.yaml"
+WORKERS=2
+DELAY="false"
+CORES=1
+POOL_SIZE="200MiB"
+DRY_RUN=
+KIND="kind"
+FALLOCATE="fallocate"
+KUBECTL="kubectl"
+DOCKER="docker"
+HUGE_PAGES=1800
+LABEL=
+
+# Function to add ZFS repository and install ZFS and LVM utilities
+install_zfs_lvm() {
+  local node=$1
+  $DOCKER exec $node bash -c '
+    set -x  # Enable debugging
+
+    # Update package lists and install prerequisites
+    apt-get update && \
+    apt-get install -y gnupg2 lsb-release || { echo "Failed to install prerequisites."; exit 1; }
+
+    # Add ZFS repository and install ZFS utilities
+    echo "deb [arch=amd64] http://deb.debian.org/debian bookworm contrib" >> /etc/apt/sources.list && \
+    curl -fsSL http://archive.zfsonlinux.org/debian/zfs-signing-key.asc | gpg --dearmor --output /etc/apt/trusted.gpg.d/zfs-archive-keyring.gpg || { echo "Failed to add ZFS repository key."; exit 1; }
+
+    apt-get update && \
+    apt-get install -y zfsutils-linux lvm2 || { echo "Failed to install ZFS and LVM utilities."; exit 1; }
+
+    # Load necessary kernel modules
+    modprobe dm_snapshot dm_thin_pool || { echo "Failed to load kernel modules."; exit 1; }
+
+    # Verify installation
+    if command -v zfs && command -v lvm; then
+        echo "ZFS and LVM utilities are installed."
+    else
+        echo "ZFS and/or LVM utilities installation failed."
+    fi
```

Comment:
This should be done in the setup-prereq script

Reply:
Moved